gcp_cost_report/main.py

```diff
@@ -185,7 +185,14 @@ def wrap_in_mrkdwn(a):
                 wrap_in_mrkdwn('\n'.join(a[0] for a in all_rows)),
                 wrap_in_mrkdwn('\n'.join(a[1] for a in all_rows)),
             ]
-            blocks = [{'type': 'section', 'fields': body}]
+
+            dashboard_message = {
+                "type": "mrkdwn",
+                "text": "For further details, visit our \
+                <https://datastudio.google.com/reporting/9f134c96-b86c-4a05-9685-4df2438537a4 | GCP Cost Dashboard>",
```

Comment:
Could you change these strings to use single quotes, please?

Reply:
Interesting link format -- do you know why it's different from the usual Markdown `[some text](https://some/link)` style?

Reply:
> Could you change these strings to use single quotes, please?

Fixed, thanks @lgruen! 

> Interesting link format -- do you know why it's different from the usual Markdown [some text](https://some/link) style?

As I understand it, mrkdwn != Markdown. Although similar, there are some differences including how hyperlinks are defined. I followed the [docs here. ](https://api.slack.com/reference/surfaces/formatting#linking-urls)